### PR TITLE
Adds multiple env support to the SB3 wrapper

### PR DESCRIPTION
**Pull request by edbeeching**
Adds an option to interact with parallel Godot executables using the `n_parallel` argument. This only works in on an export env.

Example usage with 4 parallel envs:
```
env = StableBaselinesGodotEnv(env_path=args.env_path, n_parallel=4)
```

### CODE REVIEW

**Review by visuallization — COMMENTED** [review]
Uuuuh nice, this looks really good. Also nice small improvements regarding typings, documentation and warnings. I will test this straight away and will get back to you.

**Comment by Ivan-267** [comment]
Looks great and worked properly in my quick test. 

Regarding the last commit: 
https://github.com/edbeeching/godot_rl_agents/blob/9c25e76d7d992617c39abdd6aac23b63d262ffca/examples/stable_baselines3_example.py#L28
Small note is that the help tip can be updated as well, e.g. `"How many instances of the environment executable to launch - requires --env_path to be set."`.

**Comment by edbeeching** [comment]
@Ivan-267 , good point about the seed and help description. Thanks.

File: godot_rl/wrappers/stable_baselines_wrapper.py [hunk 1]
```diff
@@ -2,55 +2,109 @@
...
+        for i in range(self.n_parallel):
+            self.envs[i].step_send(action[i*num_envs:(i+1)*num_envs])
+
+        # Receive results from each environment
+        for i in range(self.n_parallel):
+            obs, reward, term, trunc, info = self.envs[i].step_recv()
+            all_obs.extend(obs)
+            all_rewards.extend(reward)
+            all_term.extend(term)
+            all_trunc.extend(trunc)
```
Comment (araffin) [inline godot_rl/wrappers/stable_baselines_wrapper.py:61]: the truncated should be stored in info, and info should include terminal obs too, see https://github.com/DLR-RM/stable-baselines3/blob/master/stable_baselines3/common/vec_env/dummy_vec_env.py#L61-L70

Comment (edbeeching) [reply godot_rl/wrappers/stable_baselines_wrapper.py:61]: @araffin Thanks for this info. I have a question. 

On the Godot side, envs are automatically reset when an episode terminates / truncates, I am not sure how feasible it is to send the terminal obs in the truncated setting without refactoring all the envs. Does the sb3 PPO implementation use the terminal obs for a value preduction in the truncated setting? If it is not possible to refactor in the short term, would you recommend just setting the terminated flag? (with trunc = False for all steps)

**Pull request merged**